.github/workflows/build_backend.yml

```diff
@@ -0,0 +1,232 @@
+# https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow
+# https://docs.github.com/fr/actions/using-workflows/workflow-syntax-for-github-actions#exemple--inclusion-de-chemins-dacc%C3%A8s
+
+name: Full CI
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      run-integ:
+        description: 'If integ test should be run'
```

Comment:
Was "integration test" label too long for a GH UI?

Reply:
No, changing it